aries-site/src/examples/components/checkboxgroup/CheckBoxGroupSimpleExample.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { CheckBoxGroup, Form, FormField } from 'grommet';
+
+export const CheckBoxGroupSimpleExample = () => {
+  return (
+    <Form>
+      <FormField
+        name="checkbox-simple"
+        fill
+        htmlFor="simple-checkboxgroup"
```

Comment:
Wondering about this... and don't have a great answer yet. This is adding a <label> in the DOM, but the label does not have a <input> associated with it because each of the checkboxes are there own inputs with own labels.

![Screen Shot 2020-05-22 at 11 29 57 AM](https://user-images.githubusercontent.com/1756948/82697442-316ae200-9c26-11ea-8263-28ba14d3c7bd.png)


Should we be excluding htmlFor and label in the case of CheckBoxGroup?

Reply:
I've created a grommet issue about this. Grouped inputs already have their own labels and should be wrapped in a fieldset and legend to be correct. More details here: https://github.com/grommet/grommet/issues/4108

It's an important enhancement, but I think should be taken care of separately from this PR. We also need to ensure that the contained checkboxes are able to correctly associate the label with the checkbox.

Reply:
Great. Agree with moving forward with PR.